README.md

```diff
@@ -1,10 +1,24 @@
 # Lora Based BraveSensor Firmware
 
+# Project Overview
+
+## Setup
+Follow this [guide](https://docs.rakwireless.com/Product-Categories/WisDuo/RAK3272S-Breakout-Board/Quickstart/#rak3272s-breakout-board-as-a-stand-alone-device-using-rui3) on how to setup. The LoraBraveSensor directory can subsitute the Arduino Serial demo, as a guide on how to compile and upload our custom firmware. After going throught the entire process, I recommend using vscode's Arduino extension for easier file management. 
+
+## Secrets File
```

Comment:
Maybe this section should go before Setup, or be a subheading under it, because you need to have your `secrets.h` file in the right place before compiling and flashing the device, right?

Reply:
Thanks!